R/cnd-handlers.R

```diff
@@ -82,11 +85,9 @@ with_handlers <- function(.expr, ...) {
     abort("All handlers should be functions")
   }
 
-  inplace <- keep(handlers, inherits, "inplace")
-  exiting <- keep(handlers, inherits, "exiting")
-  if (length(handlers) > length(exiting) + length(inplace)) {
-    abort("all handlers should inherit from `exiting` or `inplace`")
-  }
+  is_inplace <- map_lgl(handlers, inherits, "inplace")
```

Comment:
Should we now use `as_function()` instead of `is_function()`? That way you can use the formula shortcut for exiting handlers.

Reply:
I wouldn't mind, but then should we deprecate `exiting()` since it would offer no added value (beside being extra precise)?

Reply:
I think we could leave it for symmetry with `inplace()`